README.md

```diff
@@ -38,7 +38,10 @@ Install with `npm install @octokit/core octokit-plugin-create-pull-request`. Opt
 
 ```js
 const { Octokit } = require("@octokit/core");
-const { createPullRequest } = require("octokit-plugin-create-pull-request");
+const {
+  createPullRequest,
+  deleteFile,
+} = require("octokit-plugin-create-pull-request");
```

Comment:
Seeing these two imports, I really don't like `deleteFile`. At first sight it suggests that it's a separate method to delete a file, don't you think?

Node.js has a convention to uses a `k` prefix for constants, see example: https://github.com/nodejs/node/blob/4bb06dbd0a87e5b2430780ddceec08015f8c496a/lib/_http_client.js#LL107C7-L107C13

Maybe we could use `kDeleteFile`? Or how about `DELETE_FILE`?

Reply:
Good shout. I changed it to `DELETE_FILE` which is well understood in our industry